patterns/1-initial/scoring-grid.md

```diff
@@ -0,0 +1,114 @@
+## Title
+
+Software Project Scoring Grid
+
+## Patlet
+
+Objectively improve the quality of code repositories across an organisation by introducing a Scoring Grid, which can be used to grade the state of existing code bases against InnerSource best practice.
+
+By providing a grading system, the Scoring Grid can lay out a path that engineers can follow to develop greenfield projects into mature InnerSource communities with strong ownership through incremental improvement and rescoring.
+
+## Problem
+
+Organisations are littered with repositories that don't have README files, or are outdated, containing obvious errors; projects have no CI integration, no release instructions, badly defined licenses, no PR templates, and so on. Don't let perfect be the enemy of good - software projects get started with little to no thought about their long term maintenance, and a standard plan to refine these projects and make them good is needed for the sanity and health of the engineering teams.
+
+A *Software Project Scoring Grid*, as presented in this pattern, provides a guided pathway from "new project" to a "mature, well maintained, inner source project". By grading areas such as Readme, Contribution Guidelines, Licensing, Pull Requests, and Continuous Integration; a scoring grid can objectively highlight positive and negative aspects of a project, and provide tangible guidance to engineering teams on how to make their repositories more InnerSource friendly.
```

Comment:
You reference "new project" here, and "greenfield" in L9. In our experience this pattern is just as valuable when transitioning a mature, well maintained project from closed source -> inner source when suddenly a bunch of new things become important so might be worth generalising the problem.

Reply:
Agreed, and infact I'm applying the scoring to a mature 10 year old repository at the moment - so maybe "new project" can be reworded to "new, poorly maintained, or stale project".